pkg/txm/clientwrappers/dualbroadcast/meta_client.go

```diff
@@ -0,0 +1,206 @@
+package dualbroadcast
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	evmtypes "github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+
+	"github.com/smartcontractkit/chainlink-evm/pkg/client"
+	"github.com/smartcontractkit/chainlink-evm/pkg/txm/types"
+)
+
+const (
+	timeout = time.Second * 300
+)
+
+type MetaClientKeystore interface {
+	SignMessage(ctx context.Context, address common.Address, data []byte) ([]byte, error)
+	SignTx(ctx context.Context, fromAddress common.Address, tx *evmtypes.Transaction) (*evmtypes.Transaction, error)
+}
+
+type MetaClient struct {
```

Comment:
What does `Meta` mean in this context? We are already in package `dualbroadcast`, can this just be a `dualbroadcast.Client`?

Reply:
Er, maybe just some docs would help